.github/workflows/ci.yml

```diff
@@ -6,10 +6,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Use Node.js 20.x
+      - name: Use Node.js 18.x
```

Comment:
@dmattia i had to revert this back to 18 - im basically just trying to update this package in the monorepo to take in pathfinder changes im working on but the package updates are causing all of these weird conflicts from io-ts to prettier and more.